app/utils/translation.py

```diff
@@ -4,13 +4,24 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+
+HTML_TAGS = {
+    "<strong>": "\u003cstrong\u003e",
+    "</strong>": "\u003c/strong\u003e",
+    "<br>": "\u003cbr/\u003e",
+}
+
 def load_translation(language):
     primary_language = language.split(",")[0].split(";")[0].split("-")[0]
     translations = {}
     try:
         with open(f"locales/{primary_language}.json", "r", encoding="utf-8") as f:
             translations = json.load(f)
+            for key in translations:
+                if isinstance(translations[key], str):
+                    for tag, escape_sequence in HTML_TAGS.items():
+                        translations[key] = translations[key].replace(tag, escape_sequence)
```

Comment:
This seems a bit error-prone to me, and ideally we wouldn’t need to manually match the HTML tags here! I tried commenting this out, and it still seems to load the HTML tags properly anyway? Maybe I’m missing something.

Reply:
@guadiromero Do you know more about this? Do you think we need this here? For me, the HTML tags inside the translations work fine without this extra code. 🤷

Reply:
@herrherrmann I took a look at it yesterday and I also couldn't understand the purpose of this code. Maybe we can comment it out with a TODO note to delete it later, after Urs is back and we make sure that we don't need it.

Reply:
Sounds good, I’ll do that! Thanks for checking.